Chess/state.cpp

```diff
@@ -0,0 +1,130 @@
+#include "state.h"
+
+
+State::State()
+{
+}
+
+State::~State()
+{
+}
+
+
+void State::cellAction(Game *state, QString str){ throw NotImplementedEx(); }
```

Comment:
1. Always catch exceptions by const refference
2. it's good approach to catch std::exception and CessException separtly
3. it also might be worth catching everything catch(...)
4. how to process - let's think:
a. if we have log - we can log it
b. Consider what's happened and what can we do about it.
if it's NotImplementedEx - we are requesting a wrong action on a wrong screen
we can 
- do nothing (just return)
- show something on UI - if we bother to
If it's not clear what to do with other exception - just do the same thing. maybe you will understand what is better approach to handle it when you see it happning.